.github/workflows/run-browser-tests.yml

```diff
@@ -13,6 +16,14 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      - name: Configure AWS Credentials
```

Comment:
todo: update to prod account, temporarily have this here so I can test the workflows in the test environment against this branch